### PR TITLE
Add more keyword coercers

### PR DESCRIPTION
**Pull request by mitchelkuijpers**
Adds support for:

keyword->string
keyword->number,int,long,double
keyword->boolean

Resolves: 54

### CODE REVIEW

**Comment by mitchelkuijpers** [comment]
I am not quite sure what should be added to the `json-coercions` or the `string-coercions` so please inspect that extra carefully if you can.

File: src/schema_tools/coerce.cljc [hunk 1]
```diff
@@ -236,12 +235,17 @@
       (catch #?(:clj Exception, :cljs js/Error) _ x))
     x))
 
+(defn keyword->string [x]
+  (cond-> x
+          (keyword? x) name))
```
Comment (ikitommi) [inline src/schema_tools/coerce.cljc]: this doesn't retain the namespace in qualified keywords

File: src/schema_tools/coerce.cljc [hunk 2]
```diff
@@ -4,10 +4,9 @@
             [schema.utils :as su]
             [schema.coerce :as sc]
             [schema-tools.impl :as impl]
-    #?@(:clj  [
```
Comment (ikitommi) [inline src/schema_tools/coerce.cljc]: could you reindent the lines

File: src/schema_tools/coerce.cljc [hunk 3]
```diff
@@ -236,12 +235,17 @@
...
 
+(defn keyword->string [x]
+  (cond-> x
+          (keyword? x) name))
+
 (defn collection-matcher [schema]
   (if (or (and (coll? schema) (not (record? schema))))
     (fn [x] (if (coll? x) x [x]))))
 
 (def +json-coercions+
```
Comment (ikitommi) [inline src/schema_tools/coerce.cljc]: could you add the keyword->x coercions also to json too? e.g. `{s/Int s/Int}` might come as `{:1 1}` over the wire. JSON can't represent keywords itself, so adding these keeps the coercion as strict as before, e.g. in the default case, just effects the keys.

File: src/schema_tools/coerce.cljc (hunk 2 above)
Comment (mitchelkuijpers) [reply src/schema_tools/coerce.cljc]: Should i also retain the random enter?

File: src/schema_tools/coerce.cljc (hunk 1 above)
Comment (mitchelkuijpers) [reply src/schema_tools/coerce.cljc]: Will fix good catch :+1:

**Pull request merged**